scripts/download_and_localise_package_files.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+set -o pipefail ## Pipefail, complain on new unassigned variables.
+
+VERSION='0.2.0dev'
+
+## Helptext function
+function Helptext() {
+  echo -ne "\t usage: ${0} [options] package_name\n\n"
+  echo -ne "This script takes in the name of a package as it appears in './packages', and carries out the localisation operation necessary to prepare for nf-core/eager processing.\n\n"
+  echo -ne "Options:\n"
+  echo -ne "-h, --help\t\tPrint this text and exit.\n"
+  echo -ne "-v, --version \t\tPrint version and exit.\n"
+}
+
+## Parse CLI args.
+TEMP=`getopt -q -o hv --long help,version -n "${0}" -- "$@"`
+eval set -- "${TEMP}"
+
+##Parameter defaults
+package_name=''
+script_debug_string="[localise_package_files.sh]:"
+
+## Read in CLI arguments
+while true ; do
+  case "$1" in
+    -h|--help)          Helptext; exit 0 ;;
+    -v|--version)       echo ${VERSION}; exit 0;;
+    --)                 package_name="${2}"; break ;;
+    *)                  echo -e "invalid option provided.\n"; Helptext; exit 1;;
+  esac
+done
+
+## Source helper functions
+repo_dir=$(dirname $(readlink -f ${0}))/..  ## The repository will be the original position of this script. If a user copies instead of symlink, this will fail.
+source ${repo_dir}/scripts/source_me.sh             ## Source helper functions
```

Comment:
```suggestion
repo_dir=$(dirname $(readlink -f ${0}))/..  ## The repository will be the original position of this script. If a user copies instead of symlink, this will fail.
source ${repo_dir}/scripts/source_me.sh   ## Source helper functions
```